ownCloud/Viewer/PDFViewerViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PDFViewerViewController.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 29/08/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+import PDFKit
+
+class PDFViewerViewController: DisplayViewController, DisplayExtension {
+
+	static var displayExtensionIdentifier: String = "normalPDFViewer"
```

Comment:
Let's use reverse-domain-name-style identifiers here, like f.ex. `org.owncloud.pdfViewer.default` (Michael's PDF viewer could then be added as `org.owncloud.pdfViewer.extended`, and something else as `com.othercorp.pdfViewer`).